modules/uhttp/client/client_filters_test.go

```diff
@@ -121,6 +121,44 @@ func TestExecutionChainFilters_AuthContextPropagationFailure(t *testing.T) {
 	})
 }
 
+func TestFiltersWithTracerErrors(t *testing.T) {
+	testCases := map[string]Filter{
+		"auth":    authenticationFilter(fakeAuthInfo{_testYaml}),
+		"tracing": tracingFilter(),
+	}
+
+	for name, filter := range testCases {
+		op := func(tracer opentracing.Tracer) {
+			tr := &shadowTracer{
+				tracer,
+				func(sm opentracing.SpanContext, format interface{}, carrier interface{}) error {
+					return errors.New("Very bad tracer")
+				},
+				nil,
+			}
+			opentracing.InitGlobalTracer(tr)
```

Comment:
This call returns a triplet - tracer, closer, error. You want to check for err, and close?